### PR TITLE
Fix typos and pseudo-typos 1

### PR DESCRIPTION
**Pull request by Josh-Cena**
I am fixing a lot of nits in content. The goal is to make our custom dictionaries as small as possible by eliminating things that could be better recognized as words. This not only helps with automation but helps with human readers too, especially those who may be slow to recognize words.

- About half of them are code not properly capitalized.
- There are some real typos.
- Occasionally I have to refactor the wording a bit.
- Where we are demonstrating real typos, I use inline `cSpell:ignore` to prevent it being ignored in other files.

### CODE REVIEW

**Review by chrisdavidmills — COMMENTED** [review]
@Josh-Cena I've looked through these fixes and they look OK,  but I do have one overriding concern — there may be external examples (e.g. in [dom-examples](https://github.com/mdn/dom-examples) or [django-library-tutorial](https://github.com/mdn/django-locallibrary-tutorial)) that this could cause to go out of sync with their associated guides.

Do you intend to update those too, or would it be better to leave such examples (where you are marginally improving class or variable names, for example), alone, to avoid the hassle?

**Comment by Josh-Cena** [comment]
@chrisdavidmills It would be great if the example repos can be updated too. I don't know if I have time to do that, though. Usually in this sort of batch content update, example repos often lag behind, and that wouldn't be a reason to hold back content updates, because we have well-established linting infrastructure for content but not for example repos (we totally should).

**Comment by hamishwillee** [comment]
What is the value of fixing the pseudo typos? `firefoxlogo400.png` vs `firefox-logo400.png` seems a pointless fix. If you're a reader you won't care, and as an author I will find it more annoying because selection via double click selects less of the filename.

Ditto things like "game-over" vs "gameover" - I might use game_over, or gameOver for a state name, but I would never use `game-over` as it might be tokenised to a minus sign.

All the actual typo fixes look great though.

**Comment by Josh-Cena** [comment]
> `firefoxlogo400.png` vs `firefox-logo400.png` seems a pointless fix. If you're a reader you won't care, and as an author I will find it more annoying because selection via double click selects less of the filename.

As a reader, a well-separated name is slightly easier to scan. If ease of selection is a showstopper for you, I'm happy to change it to an underscore. From a maintenance perspective, using valid words eliminates the chance of false-negative spell checking elsewhere since we don't have to exempt "firefoxlogo".

> Ditto things like "game-over" vs "gameover" - I might use game_over, or gameOver for a state name, but I would never use `game-over` as it might be tokenised to a minus sign.

Here it's not a code entity though. I could be wrong but I searched everywhere and there's no reference to `gameover` in code, so I assume it's meant to be plain-English description of the code state, in which case we would either say "game-over" or "game over", and I chose the former because it looks slightly more correct when modifying a noun.

File: files/en-us/games/techniques/control_mechanisms/desktop_with_gamepad/index.md [hunk 1]
```diff
@@ -346,7 +346,7 @@ if (gamepadAPI.buttonPressed("B")) {
 }
 ```
 
-## The paused and gameover states
+## The paused and game-over states
 
 We already learned how to control the whole lifecycle of the game: pausing the gameplay, restarting it, or getting back to the main menu. It works smooth on mobile and desktop, and adding gamepad controls is just as straightforward — in the `update()` function, we check to see if the current state status is `paused` — if so, the relevant actions are enabled:
```
Comment (hamishwillee) [inline files/en-us/games/techniques/control_mechanisms/desktop_with_gamepad/index.md:351]:
```suggestion
We already learned how to control the whole lifecycle of the game: pausing the gameplay, restarting it, or getting back to the main menu. It works smooth on mobile and desktop, and adding gamepad controls is just as straightforward — in the `update()` function, we check to see if the current state status is "paused" — if so, the relevant actions are enabled:
```

**Review by hamishwillee — APPROVED** [review]
Thanks @Josh-Cena - you've clearly been very careful. Appreciated.

**Pull request merged**